.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ jobs:
       - checkout
       - run:
           name: Install dependencies
-          command: yarn install --frozen-lockfile
+          command: yarn install
```

Comment:
why you remove frozen?

Reply:
> why you remove frozen?

CircleCI is [throwing this error](https://circleci.com/gh/pesto-students/batch-10-tripen/176?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-
) because `--frozen-lockfile` stops from updating / adding dependency versions in yarn.lock file

![Screenshot_2019-09-24_07-51-21](https://user-images.githubusercontent.com/10175625/65476444-9b1ccb80-dea0-11e9-90a2-099265fb7218.png)